src/graphql-server/schema.graphql

```diff
@@ -32,13 +32,51 @@ type HostPricing {
   pricePerUnit: String
 }
 
+enum TransactionType {
+  offer
+  request
+}
+
+enum TransactionStatus {
+  pending
+  complete
+  rejected
+}
+
+enum TransactionDirection {
+  incoming # indicates the recipient of funds
+  outgoing # indicates the spender of funds
+}
+
+type Transaction {
+  id: ID # ID will be the originCommitHash
+  amount: Float
+  counterparty: String
+  direction: TransactionDirection
+  status: TransactionStatus
+  type: TransactionType
+  timestamp: String # QUESTION: Can we use GraphQLDateTime here instead? This timestamp is actually received as an ISO8601...
```

Comment:
It's probably right to use the most specific type available. I tend to get lazy and just use String for everything but let's try the more correct way.

Reply:
Let's hold on to this one to refer to soon. In order to reference that type, we need to import a specific module, which is causing some errors.